docs/components/modeler/using-web-and-desktop-modeler-together.md

```diff
@@ -0,0 +1,80 @@
+| ----------------------------------------------------- | ------------------------------------ |
+| Can load multiple templates defined in a single file. | Only supports one template per file. |
+
+To maintain compatibility, avoid defining multiple templates in a single file.
+
+### Shared element templates
+
+| Desktop Modeler                                                                                                                                    | Web Modeler                                                                                                                                                         |
+| -------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Templates can be installed as [global templates](/components/modeler/desktop-modeler/element-templates/configuring-templates.md#global-templates). | Templates are [published to the organization](/components/connectors/manage-connector-templates.md#manage-published-connector-templates) for reuse across projects. |
+
+Camunda recommends storing shared templates in a separate VCS repository:
+
+- **Desktop Modeler**: Copy templates manually into your global directory.
+- **Web Modeler**: Use a CI/CD pipeline to sync templates with your repository via the [Web Modeler API](/apis-tools/web-modeler-api/index.md).
+
+### Project element templates
+
+| Desktop Modeler                                                                                                                                                                                          | Web Modeler                                                                                                                                  |
+| -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------- |
+| Discovers [local element templates](/components/modeler/desktop-modeler/element-templates/configuring-templates.md#global-templates) are loaded from the `.camunda/element-templates` folder if present. | Processes templates from a single folder. <br></br>If working with a project (not a process application), templates must first be published. |
+
+:::note
+
+- If starting in **Desktop Modeler**, use a single folder for your process application. This makes project templates available in both modelers without extra work.
+- If starting in **Web Modeler**, manually create an empty `.process-application` file after cloning the repository so Desktop Modeler can correctly recognize the project.
```

Comment:
I'd suggest to create an empty JSON object in the file `{}` as we may store some data there in the future. An empty file will cause parsing error.